src/Main.java

```diff
@@ -0,0 +1,166 @@
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Main obj = new Main();
+
+
+
+        System.out.println("Метод findSymbolOccurance");
+
+        obj.findSymbolOccurance("Java", 'a');
+
+        System.out.println();
+
+        System.out.println("Метод findWordPosition");
+
+
+        obj.findWordPosition("Apollo", "pollo" );
+        obj.findWordPosition("Aplle", "Plant" );
+
+        System.out.println();
+
+        System.out.println("Метод stringReverse");
+        obj.stringReverse("Hello");
+
+        System.out.println();
+
+        System.out.println("Метод isPalindrome");
+        obj.isPalindrome("ERE");
+        obj.isPalindrome("Allo");
+
+        System.out.println();
+        System.out.println();
+
+        System.out.println("Гра - вгадай слово");
+        obj.guessWord();
+
+
+
+
+
+
+
+
+    }
+
+    public void findSymbolOccurance (String str, char ch) {
+
+        char[] charArray = str.toCharArray();
+        int count = 0;
+
+        for ( int i = 0; i < charArray.length; i++ ) {
+            if ( charArray[i] == ch ) {
+                count++;
+            }
+        }
+
+        System.out.println("Символ зустрічається : " + count + " раз/разів");
+    }
+
+
+    public void findWordPosition (String source, String target) {
+
+        if (source.contains(target)) {
+            System.out.println("Output : " + "1");
+        } else {
+            System.out.println("Output : " + "-1");
+        }
+    }
+
+
+    public void stringReverse (String simpleString) {
+
+        char[] arrayChar = simpleString.toCharArray();
+        String result = "";
+
+        for ( int i = arrayChar.length - 1; i >= 0; i--) {
+            result += String.valueOf(arrayChar[i]);
+        }
+        System.out.println(result);
+
+    }
+
+
+    public void isPalindrome (String checkPalindrome) {
+
+        String lower = checkPalindrome.toLowerCase();
+        char[] tempArray = lower.toCharArray();
+        String reverseWord = "";
+
+        for ( int i = tempArray.length - 1; i >= 0; i--) {
+            reverseWord += String.valueOf(tempArray[i]);
+        }
+
+        if (lower.equals(reverseWord)) {
+            System.out.println("True");
```

Comment:
Please remove System.out.println() statements. Method isPalindrome() should return a boolean. Please refactor it.